src/model/Model_Attachment.cpp

```diff
@@ -25,10 +25,10 @@ const std::vector<std::pair<Model_Attachment::REFTYPE, wxString> > Model_Attachm
     {Model_Attachment::STOCK, wxTRANSLATE("Stock")},
     {Model_Attachment::ASSET, wxTRANSLATE("Asset")},
     {Model_Attachment::BANKACCOUNT, wxTRANSLATE("BankAccount")},
-    {Model_Attachment::BILLSDEPOSIT, wxTRANSLATE("ScheduledTransaction")},
+    {Model_Attachment::BILLSDEPOSIT, wxTRANSLATE("RecurringTransaction")},
     {Model_Attachment::PAYEE, wxTRANSLATE("Payee")},
     {Model_Attachment::TRANSACTIONSPLIT, wxTRANSLATE("TransactionSplit")},
-    {Model_Attachment::BILLSDEPOSITSPLIT, wxTRANSLATE("ScheduledTransactionSplit")}
+    {Model_Attachment::BILLSDEPOSITSPLIT, wxTRANSLATE("RecurringTransactionSplit")}
```

Comment:
I don't like this piece of code.
translators will ask questions why there are no spaces.

there are none, it seems that there are no spaces in the attachments in the file path.
where it's just an algorithm to change spaces to _.

Reply:
There has been no change with this patch as it has been reverted.